app/Http/Controllers/PagesController.php

```diff
@@ -63,4 +63,33 @@ public function help()
     {
         return view('help');
     }
+
+    /**
+     * @return comments.blade.php
+     */
+    public function comments()
+    {
+        $projects = Project::orderBy('created_at', 'desc')->paginate(12);
+
+        return view('pages.comments', ['projects' => $projects]);
+    }
+
+    /**
+     * @return views.blade.php
+     */
+    public function views()
+    {
+       $projects = Project::orderBy('views', 'desc')->paginate(12);
+
+        return view('pages.views', ['projects' => $projects]);
+    }
+    /**
+     * @return likes.blade.php
+     */
+    public function likes()
+    {
+        $projects = Project::orderBy('likes', 'desc')->paginate(12);
+
+        return view('pages.likes', ['projects' => $projects]);
```

Comment:
Because each of the pages reflect either most viewed or most liked. They are different criteria.